maintenance/extract_data/extract_data.py

```diff
@@ -54,7 +54,7 @@ def db_to_csv(connection, query):
 
 def get_extract_zip_file_path(current_time):
     base_dir = Path(os.environ['EXTRACTS_DIRECTORY'])
-    file_name = f"data-extract-{current_time:%Y-%m-%d}.zip"
+    file_name = f"data-extract-{current_time:%Y-%m-%d-%H_%M_%S}.zip"
```

Comment:
The time is included in the filename to avoid overwriting a file, since that would create two database records pointing to the same file.
While the above should fix the problem because I don't envision two extracts being created in the same second, I'm thinking it might actually be worth throwing an error if we're trying to overwrite a file that already exists, because having two extract records with the same file is a bad situation

Reply:
Including timestamp looks good; yes let's throw an error.